yucca/training/configuration/configure_plans.py

```diff
@@ -26,14 +26,20 @@ def lm_hparams(self):
         }
 
 
-def get_plan_config(path_config: PathConfig, continue_from_most_recent: bool):
-    plans = setup_plans(
-        path_config.ckpt_path,
-        continue_from_most_recent,
-        path_config.plans_path,
-        path_config.version,
-        path_config.version_dir,
-    )
+def get_plan_config(plans_path: str, stage: Literal["fit", "test", "predict"], ckpt_plans: Union[dict, None] = None, continue_from_most_recent: bool):
+    # First try to load torch checkpoints and extract plans and carry-over information from there.
+    if stage == "fit":
+        plans = load_plans(plans_path)
+    elif stage == "test":
+        raise NotImplementedError
+    elif stage == "predict":
+        # In this case we don't want to rely on plans being found in the preprocessed folder,
+        # as it might not exist.
+        assert ckpt_plans is not None
+        plans = ckpt_plans
+    else:
+        raise NotImplementedError(f'Stage: {stage} is not supported')
```

Comment:
I would just make an assert as the first line, and then we know that this if statement will always catch one case:

```python
assert stage in ["fit", "test", "predict"], f'stage: {stage} is not supported'
```